src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  &__title {
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 32px;
+    letter-spacing: -1.5px;
+    text-align: center;
+    margin-bottom: 48px;
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__input {
+    width: 100%;
+    padding: 14px 16px;
+    margin: 16px 0;
+    border-radius: 40px;
+    border: none;
+    background: $text-color-light;
+    cursor: pointer;
+
+    &:first-child {
+      margin: 0;
+    }
+
+    &::placeholder {
+      transition: color 0.4;
+      color: #7e7e83;
+      font-size: 14px;
+      line-height: 20px;
+    }
+
+    &:hover::placeholder {
+      color: #292929;
+    }
+  }
+
+  &__message {
+    display: flex;
+    width: 100%;
+    height: 148px;
+    padding: 14px 16px;
+    border: none;
+    margin-bottom: 32px;
+    background: $text-color-light;
+    border-radius: 16px;
+    resize: none;
+    cursor: pointer;
+    transition: color 0.4s;
+
+    &:hover::placeholder {
+      color: #292929;
+    }
+  }
```

Comment:
input and message have a lot of things in common
You should have made a common style for all form elements and re-use it, while message and input would held only diferences